### PR TITLE
Updating dial-out to use Long virtual number as FROM

### PR DESCRIPTION
**Pull request by javiermolsanz**

### CODE REVIEW

File: Dial-Out/config.example.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ module.exports = {
     apiKey: '',
     apiSecret: '',
     sip: {
+        from: '',
         username: '',
         password: '',
     },
-};
\ No newline at end of file
+};
```
Comment (anandpaithankar1) [inline Dial-Out/config.example.js:9]: Please add a `newline` at the end of these files.

**Comment by anandpaithankar1** [comment]
Side note - This application already works with Nexmo as a SIP provider without providing any valid `from` number. Nexmo also uses SIP based authentication using supplied Nexmo API key and value as SIP username and password. I understand this PR is in relation with the article, but it is not mandatory to have LVN.  Our customers also refer this application very often, so I don't want to using LVN as a mandatory step adding to confusion.

**Comment by javiermolsanz** [comment]
> I'd rewrite this paragraph as follows -
> 
> * Sign up for a [Nexmo](https://www.nexmo.com/) account to get the API Key and Secret.
> 
> Optionally, you can also rent a Long Virtual Number (LVN) from Nexmo or your preferred provider to use it for Call Line Identifier (CLI). This LVN can be set in `from` field to be used as a CLI.
> Although, this step is not mandatory, however, the downstream carriers may reject the call if the correct CLI has not been specified as explained in [this article](https://help.nexmo.com/hc/en-us/articles/204015273-Does-Nexmo-support-CLI-Caller-ID-for-my-voice-calls-#:~:text=When%20using%20any%20of%20the,as%20your%20CLI%2FCaller%20ID.&text=CLI%20for%20in%2Dcountry%20originating,all%20of%20our%20virtual%20numbers.)

Sounds good to me :)

**Review by anandpaithankar1 — APPROVED** [review]

**Comment by anandpaithankar1** [comment]
Thank you for improving.

**Review by msach22 — APPROVED** [review]
LGTM

**Pull request merged**